app/src/renderer/scripts/num.js

```diff
@@ -59,10 +59,27 @@ function uatoms(number = 0) {
   return BigNumber(number).times(1e6).toString()
 }
 
+// convert micro denoms like uatom to display denoms like atom
+function denom(denom) {
+  if (denom.charAt(0) === `u`) {
+    return denom.substr(1)
+  }
+  return denom
+}
+
+function viewCoin({ amount, denom: _denom }) {
```

Comment:
denom has been renamed _denom for the upper function name clashing?

Reply:
yes, should be named differently?